Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -53,14 +57,52 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
        <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
        for instructions on how to install ``az`` and prepare your account.
 
-       .. code-block:: shell-session
+       .. code-block:: bash
```

Comment:
@qmonnet If I leave this with `shell-session` I can't keep the comments on lines 66-68 for example.

Reply:
Makes sense to me to use `bash` here, it's still the best thing to use when the snippet is long enough and looks like a script more like a couple of commands fro the console. Plus this one does not have any output from the commands anyway. I'm not opposed to `bash` in general if that's the most relevant choice.

(Having occasional comments in `shell-session` blocks is tricky, using `$ # This is my comment...` is probably the best we have but does not look ideal. Too bad comments and root prompt use the same symbol :/. Anyway, clearly in your case that would be cumbersome and using `bash` looks best.)

Reply:
@qmonnet would it make sense for us to just ignore the `#` for prompts? I would be OK with just having `$`, even for root prompts, and having to specify when something requires privileges.